### PR TITLE
VSync Configuration

### PR DESCRIPTION
**Pull request by keyosk**
Full credit to Adam for this improvement: https://github.com/runelite/runelite/commit/13ecfc0a9eaf3161b979f6e9f1b6f66fa075fb51

### CODE REVIEW

**Comment by xxEzri** [comment]
Also worked on (discussion, direction, testing) by Aeryn, breakthetargets, Hooder, sosodev & maybe more on 117 and runelite discords

**Comment by keyosk** [comment]
> Also worked on (discussion, direction, testing) by Aeryn, breakthetargets, Hooder, sosodev & maybe more on 117 and runelite discords

Great shout outs. Thanks to everyone for this collaboration. 🙌

**Comment by sosodev** [comment]
I've been testing with these changes and they're kind of amazing. Performance is so much more stable on my system with: unlocked FPS + no vsync + 60 FPS target. Thank you Adam for your excellent work here and Keyosk for porting these upgrades.

File: src/main/java/rs117/hd/HdPlugin.java [hunk 1]
```diff
@@ -2188,11 +2184,12 @@ public void onConfigChanged(ConfigChanged event)
 				recompileProgram();
 				break;
 			case "unlockFps":
-				configUnlockFps = config.unlockFps();
+			case "vsyncMode":
+			case "fpsTarget":
 				clientThread.invokeLater(() ->
```
Comment (Adam-) [inline src/main/java/rs117/hd/HdPlugin.java:2189]: This invokeLater is unnecessary

Comment (keyosk) [reply src/main/java/rs117/hd/HdPlugin.java]: Right you are 🙌 Thanks for the feedback. I've addressed that here: https://github.com/RS117/RLHD/pull/181/commits/e1f3b4a4efd0e782fdee2ae42ee6e036cfbf3737

**Pull request merged**